packages/api/cms-api/src/dam/blocks/dam-file-download-link-block-transformer.service.ts

```diff
@@ -34,13 +34,15 @@ export class DamFileDownloadLinkBlockTransformerService implements BlockTransfor
                 name: file.name,
                 fileUrl: await this.filesService.createFileUrl(file, { previewDamUrls, relativeDamUrls }),
                 size: Number(file.size),
+                scope: file.scope,
```

Comment:
Why is this needed? And if it's needed, it probably should be in the block meta and handled in `output2State` correctly.

Reply:
It's necessary because when copying a page to another scope, all files on the page are also copied if they do not exist in the scope. For this check, the file scope is necessary (see https://github.com/vivid-planet/comet/blob/356ba1737f017325525823bbffba640910bea56f/packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages/sendPages.tsx#L149-L150)

I did it the same way it's done for the PixelImageBlock (https://github.com/vivid-planet/comet/blob/e1592759458bb9c9b47b1eb3e997bf9a0cb40cc8/packages/api/cms-api/src/dam/blocks/pixel-image-block-transformer.service.ts#L62)